azurelinuxagent/common/cgroup.py

```diff
@@ -117,23 +118,22 @@ def is_active(self):
                                  ' Internal error: {1}'.format(self.path, ustr(e)))
         return False
 
-    def get_tracked_metrics(self):
+    def get_tracked_metrics(self, **_):
         """
         Retrieves the current value of the metrics tracked for this cgroup and returns them as an array
         """
         raise NotImplementedError()
 
 
 class CpuCgroup(CGroup):
-    def __init__(self, name, cgroup_path, track_throttled_time=False):
```

Comment:
There is no need to maintain this flag for each instance of the CpuCgroup currently tracked need to use the same value. Moved the flag to CGroupsConfigurator.